src/main/java/org/sputnikdev/bluetooth/gattparser/spec/BluetoothGattSpecificationReader.java

```diff
@@ -77,8 +75,9 @@
 
     private static final FilenameFilter XML_FILE_FILTER = (dir, name) -> name.toLowerCase().endsWith(".xml");
```

Comment:
I think we no longer need it. It was used to load files from classpath, but we no longer do it. Some legacy code that should be removed...

Reply:
ok, removed all unused code I've found